website/docs/Pipelines/SlideSeq_Pipeline/README.md

```diff
@@ -0,0 +1,235 @@
+
+To search releases of this and other pipelines, use the WARP command-line tool [Wreleaser](https://github.com/broadinstitute/warp/tree/master/wreleaser).
+
+If you’re running a SlideSeq workflow version prior to the latest release, the accompanying documentation for that release may be downloaded with the source code on the WARP [releases page](https://github.com/broadinstitute/warp/releases) (see the source code folder `website/docs/Pipelines/SlideSeq_Pipeline`.
+
+The SlideSeq pipeline can be deployed using [Cromwell](https://cromwell.readthedocs.io/en/stable/), a GA4GH compliant, flexible workflow management system that supports multiple computing platforms. The workflow can also be run in [Terra](https://app.terra.bio), a cloud-based analysis platform. 
+
+<!--- link to public workspace will go here --->
+
+### Inputs
+
+The SlideSeq workflow inputs are specified in JSON configuration files. Example configuration files can be found in the [test_inputs](https://github.com/broadinstitute/warp/tree/master/pipelines/skylab/slide_seq/test_inputs) folder in the WARP repository.
+
+#### Input descriptions
+
+| Parameter name | Description | Type |
+| --- | --- | --- | 
+|  r1_fastq | Array of Read 1 FASTQ files; forward reads containing bead barcodes and Unique Molecular Identifiers (UMIs). | Array[File] |
+| r2_fastq | Array of Read 2 FASTQ files; reverse reads containing the cDNA fragment generated from captured mRNA. | Array[File] |
+| i1_fastq | Optional array of i1 (index) FASTQ files; index reads used for demultiplexing of multiple samples on one flow cell. | Array[File] | 
+| input_id | Name of sample matching this file; inserted into read group header. | String |
+| read_structure | Description of the UMI (M) and Barcode (C) positions in the Read 1 FASTQ. | String |
+| tar_star_reference | Cloud path to the TAR file containing a species-specific reference genome and GTF; generated using the [BuildIndices workflow](https://github.com/broadinstitute/warp/tree/develop/pipelines/skylab/build_indices/BuildIndices.wdl). | File | 
+| annotations_gtf | Cloud path to the GTF containing gene annotations used for gene tagging (must match GTF in STAR reference). | File | 
+| whitelist | Cloud path to the list of bead barcodes created by sequencing prior to mRNA transfer and library preparation. | String |
+| output_bam_basename | Optional string used for the output BAM file basename; default is set to “input_id”. | String | 
+| count_exons | Optional boolean indicating if the workflow should calculate exon counts; default is set to “false” and produces a Loom file with whole-gene counts; when set to “true” an additional layer of the Loom file is produced with exon counts. | Boolean |
+| bead_locations | Locations (XY coordinates) of the sequenced beads on the puck. | File |
```

Comment:
We decided to make bead_locations and whitelist as a single input file. It is named whitelist in the pipeline and it is a tsv file with bead barcodes, x coordinates and y coordinates for each bead in a  single line.

Reply:
My bad! I misunderstood and I must have been looking at the wrong branch. It looks like fk_cloud_slideseq has been updated most recently. Is that right?

Reply:
Updated